app/src/shared/utils/time.js

```diff
@@ -38,15 +40,18 @@ function format(diff, divisor, unit, prev) {
     return val <= 1 ? prev : `${val} ${unit}s ago`
 }
 
-export function ago(date: Date) {
+export function ago(date: Date, isTitleized?: boolean) {
```

Comment:
Is it possible to leave `titleize` out from this function altogether and use just use it as needed, eg.:

```js
titleize(ago(new Date(stream.lastUpdated)))
```

Reply:
> You could add some ellipses to long names while you are at it

Ah, thanks forgot about those. 


> Is it possible to leave `titleize` out from this function altogether and use just use it as needed, eg.:
> 
> ```js
> titleize(ago(new Date(stream.lastUpdated)))
> ```

Yep, I can do that.